target_chains/cosmwasm/contracts/pyth/src/contract.rs

```diff
@@ -182,6 +197,14 @@ fn is_allowed_tx_fees_denom(deps: &Deps, denom: &String) -> StdResult<bool> {
 fn is_fee_sufficient(deps: &Deps, info: MessageInfo, data: &[Binary]) -> StdResult<bool> {
     let state = config_read(deps.storage).load()?;
 
+    // as with chains other than osmosis
+    // if the base fee amount is set to 0
+    // no funds are required
+    // here also the behavior is same
+    if state.fee.amount.u128() == 0 {
+        return Ok(true);
+    }
```

Comment:
why do you need this special case for 0 here and above? it seems like the logic should work without it